pkg/mls/store/queries/queries.sql.go

```diff
@@ -404,9 +405,9 @@ func (q *Queries) InsertInboxLog(ctx context.Context, arg InsertInboxLogParams)
 
 const insertWelcomeMessage = `-- name: InsertWelcomeMessage :one
 SELECT
-	id, created_at, installation_key, data, hpke_public_key, installation_key_data_hash, wrapper_algorithm
+	id, created_at, installation_key, data, hpke_public_key, installation_key_data_hash, wrapper_algorithm, message_cursor
 FROM
-	insert_welcome_message_v2($1, $2, $3, $4, $5)
+	insert_welcome_message_v3($1, $2, $3, $4, $5, $6)
```

Comment:
This is a generated file, and I don't see a corresponding change in `queries.sql`. Was this hand edited?

That would work for this PR, but the next time somebody generates the queries, they would unknowingly break everything